optimism/src/keccak/mod.rs

```diff
@@ -34,7 +33,7 @@ pub(crate) type E<F> = Expr<ConstantExpr<F>, KeccakColumn>;
 pub enum Error {
     Constraint(Constraint),
     #[allow(dead_code)]
```

Comment:
Is it still required?

Reply:
Oh! Probably not anymore, I will fix it!